west.yml

```diff
@@ -97,7 +96,11 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 3eed8cbd8f6b0fa38aaa6b7d5d0e33bd7408f0d1
+      revision: pull/177/head
```

Comment:
I believe you should update `sdk-zephyr` as well?

Reply:
fixed,